lib/k8s-e2e-tests/tests/vector-dd-agent-aggregator.rs

```diff
@@ -0,0 +1,174 @@
+use indoc::indoc;
+use k8s_e2e_tests::*;
+use k8s_test_framework::{lock, test_pod, vector::Config as VectorConfig};
+use serde_json::Value;
+
+const HELM_CHART_VECTOR_AGGREGATOR: &str = "vector-aggregator";
+
+const HELM_VALUES_DDOG_AGG_TOPOLOGY: &str = indoc! {r#"
+    service:
+      type: "ClusterIP"
+      ports:
+        - name: datadog
+          port: 8080
+          protocol: TCP
+          targetPort: 8080
+    sources:
+      datadog-agent:
+        type: "datadog_logs"
+        rawConfig: |
+          address = "0.0.0.0:8080"
```

Comment:
```suggestion
        address: "0.0.0.0:8080"
        rawConfig: |
          address = "0.0.0.0:8080"
```

Since rawConfig is "deprecated" we may just want to use the newer style here